service/service.go

```diff
@@ -138,6 +139,20 @@ func New(options ...Option) (Owner, error) {
 		metrics.Freeze()
 	}
 
+	if svc.Tracer() == nil {
+		svc.configProvider.GetValue("tracing").PopulateStruct(&svc.tracerConfig)
```

Comment:
Not sure how lint passed, but you need to check for err returned on PopulateStruct